src/modules/message.module.js

```diff
@@ -0,0 +1,43 @@
+import { Module } from '../core/module';
+import { random } from '../utils';
+
+export class MessageModule extends Module {
+
+    static #messagesArray = ['Have a nice weekend!', 'Happy Thanksgiving Day!', 'All the leaves are brown)', 'Id be safe and warm'];
+
+    constructor(type, text) {
+        super(type, text);
+    };
+
+    trigger() {
+        try {
+            const body = document.querySelector('body');
+            const messageBlock = this.createMessageElement();
+            body.append(messageBlock);
```

Comment:
You could use instead of it and row in prewious comment - document.body

Reply:
done as recommended